app-template/config-template.xml

```diff
@@ -95,6 +95,7 @@
       <variable name="THIRD_URL_SCHEME" value="bitcoincash" />
       <variable name="FOURTH_URL_SCHEME" value="bchtest" />
       <variable name="FIFTH_URL_SCHEME" value="ethereum" />
+      <variable name="SIXTH_URL_SCHEME" value="ripple" />
```

Comment:
We need to add this variable to the plugin: https://github.com/cmgustavo/Custom-URL-scheme 
It's very simple.

Reply:
Done. https://github.com/cmgustavo/Custom-URL-scheme/releases/tag/v8.0.0